lib/pbench/server/api/resources/endpoint_configure.py

```diff
@@ -0,0 +1,145 @@
+                is "v5.result-data."
+            run_index: The "master" run-data index root. In the current ES
+                schema, this is "v6.run-data."
+            run_toc_index: The Elasticsearch V7 index for run TOC data. In
+                the current ES schema, this is "v6.run-toc."
+        api:    A list of the server APIs supported; we give a name, which
+                identifies the service, and the full URI relative to the
+                configured host name and port (local or remote reverse proxy)
+            results: Direct access to Pbench data sets for the dashboard; this
+                is currently direct HTTP access through Apache public_html, and
+                not an "API" as such; we will need to adopt this as a real
+                server API when Pbench moves from local filesystem to S3 storage
+                schemas
+            elasticsearch: The Pbench pass-through URI for the Elasticsearch
+                cluster. This will eventually be superceded by the native
+                Pbench APIs, though it might remain accessible with special
+                user/group privileges to support special cases
+            graphql: The GraphQL frontend on postgreSQL currently used by the
+                dashboard user mocks. This will be superceded and deprecated
+                by native Pbench user management APIs
+            queryControllers: Return information about the run documents that
+                were created within a specified range of months.
+            queryMonthIndices: Return the YYYY-mm date strings for all ES
+                vx.run-data.* indices, in descending order
+
+        TODO: We need an internal mechanism to track the active versions of the
+        various Elasticsearch template documents. We're hardcoding them here and
+        in other APIs. We should consider persisting an equivalent of the
+        mapping table built in "indexer.py" for use across the server APIs.
+
+        TODO: We provide Elasticsearch index root names here, which the dashboard
+        code needs to perform the queries we've not yet replaced with server-side
+        implementations. The entire "indices" section can be removed once that is
+        resolved.
+        """
+        self.logger.info("Received these headers: {!r}", request.headers)
+        header = request.headers.get("Forwarded")
+        origin = None
+        if header:
+            m = self.forward_pattern.search(header)
+            if m:
+                origin = m.group("host")
+                self.logger.info("Forwarded: {}, ({})", header, origin)
+        if not origin:
+            header = request.headers.get("X-Forwarded-Host")
+            if header:
+                m = self.x_forward_pattern.search(header)
+                if m:
+                    origin = m.group("host")
+                    self.logger.info("X-Forwarded-Host: {}, ({})", header, origin)
```

Comment:
Can we consider making this `.info()` message, and its sister at line 96 `.debug()`?

Reply:
They were there for debugging when I pushed queries, and I'd meant to remove them. I do want some way to reconstruct the forwarding headers in case something goes wrong, and I want that to be info so we can debug from a log rather than trying to reproduce it.

Reply:
Why not just add this information to the final `.info()` message below and drop these entirely?